airflow/providers/amazon/aws/operators/sagemaker.py

```diff
@@ -666,6 +660,12 @@ def expand_role(self) -> None:
             config["ExecutionRoleArn"] = hook.expand_role(config["ExecutionRoleArn"])
 
     def execute(self, context: Context) -> dict:
+        if self.action_if_job_exists not in ("increment", "fail", "timestamp"):
+            raise AirflowException(
+                f"Argument action_if_job_exists accepts only 'timestamp', 'increment' and 'fail'. \
+                Provided value: '{self.action_if_job_exists}'."
+            )
+
```

Comment:
With this change, instead of warning the user in dev time to fix the param, you move it to runtime. Personally I prefer to keep it in init :thinking: